File: CPP/PetShop.cpp
#include <bits/stdc++.h>

using namespace std;

class PetShop{
private:
    int id;
    string nama;
    double harga;
    int stok;

public:
    // Constructor
    PetShop(){
    }

    PetShop(int id, string nama, double harga, int stok){
        this->id = id;
        this->nama = nama;
        this->harga = harga;
        this->stok = stok;
    }

    // Setter dan Getter

    // Setter id
    void setId(int id){
        this->id = id;
    }

    // Setter nama produk
    void setNama(string nama){
        this->nama = nama;
    }

    // Setter stok
    void setStok(int stok){
        this->stok = stok;
    }

    // Setter harga
    void setHarga(double harga){
        this->harga = harga;
    }

    // Getter id
    int getId(){
        return id;
    }

    // Getter nama produk
    string getNama(){
        return nama;
    }

    // Getter stok
    int getStok(){
        return stok;
    }

    // Getter harga
    double getHarga(){
        return harga;
    }

    ~PetShop(){
    }
};

File: Python/PetShop.py
class PetShop:
    def __init__(self, id=None, nama=None, harga=None, stok=None):
        self.id = id
        self.nama = nama
        self.harga = harga
        self.stok = stok

    # setter dan getter

    # setter id
    def set_id(self, id):
        self.id = id

    # setter nama produk
    def set_nama(self, nama):
        self.nama = nama

    # setter stok
    def set_stok(self, stok):
        self.stok = stok

    # setter harga
    def set_harga(self, harga):
        self.harga = harga

    # getter id
    def get_id(self):
        return self.id

    # getter nama produk
    def get_nama(self):
        return self.nama

    # getter stok
    def get_stok(self):
        return self.stok

    # getter harga
    def get_harga(self):
        return self.harga